modules/doc/content/getting_started/installation/conda.md

```diff
@@ -1,86 +1,23 @@
 # Conda MOOSE Environment
 
-Our preferred method for obtaining libraries necessary for MOOSE based Application development, is via Conda's myriad array of libraries. Follow these instructions to create an environment on your machine using Conda. At this time, Windows is not supported. Effort is being made towards this direction. But it is some time out.
+Our preferred method for obtaining libraries necessary for MOOSE based
+Application development, is via Conda's myriad array of libraries. Follow these
```

Comment:
Is BISON being tested against CONDA?

Reply:
I'm not sure about BISON specifically, but MOOSE is (although Jason will know).  The old MOOSE package links that we linked to in BISON are no longer available, so the MOOSE team recommended updating the getting started instructions in BISON.

Reply:
Yes, BISON is tested against conda on the Mac test of the civet recipe